src/wp-includes/class-wp-html-text-replacement.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * HTML Tag Processor: Text replacement class.
+ *
+ * @package WordPress
+ * @subpackage HTML
+ * @since 6.2.0
+ */
+
+if ( ! class_exists( 'WP_HTML_Text_Replacement' ) ) :
```

Comment:
Remove this guard. See https://github.com/WordPress/wordpress-develop/pull/3920/files#r1090712903 for reasoning.